cli/tests/pcluster/config/test_section_fsx.py

```diff
@@ -173,6 +168,26 @@ def test_fsx_section_to_cfn(mocker, section_dict, expected_cfn_params):
         ("per_unit_storage_throughput", "100", 100, None),
         ("per_unit_storage_throughput", "200", 200, None),
         ("per_unit_storage_throughput", "101", 101, "'per_unit_storage_throughput' has an invalid value '101'"),
+        ("daily_automatic_backup_start_time", None, None, None),
+        ("daily_automatic_backup_start_time", "", "", "'daily_automatic_backup_start_time' has an invalid value ''"),
+        ("daily_automatic_backup_start_time", "01:00", "01:00", None),
+        ("daily_automatic_backup_start_time", "23:00", "23:00", None),
+        (
+            "daily_automatic_backup_start_time",
+            "25:00",
+            "25:00",
+            "'daily_automatic_backup_start_time' has an invalid value '25:00'",
+        ),
```

Comment:
This may be a bit pedantic, but can you include a case where the `:` is omitted? From the regex in the mappings file it looks like that's a valid format.